CONFIGURING_CLION.md

```diff
@@ -0,0 +1,24 @@
+# Configuring Clion for Rust-only development
+These are the steps to build the application-services desktop in clion:
```

Comment:
"These are steps required to build application-services Rust components in JetBrains CLion..."

Link to CLion using markdown hyperlinks: `[text](https://url)`